pi/Dockerfile

```diff
@@ -43,8 +40,6 @@ ENV SIA_MODULES gctwhr
 COPY --from=zip_downloader /sia/siac /sia/siad /usr/bin/
 COPY scripts/*.sh ./
 
-EXPOSE 9980 9981 9982
-
-HEALTHCHECK --interval=10s CMD ["./healthcheck.sh"]
+EXPOSE 9980 9981 9982 9983 9984
```

Comment:
why do we expose anything else than 9980 ?

Reply:
We are using all of these in `siad`. I'm not sure if we should be exposing them. Maybe not? Correct me if I'm wrong but we can always expose additional ports with command line arguments, right? If that's the case then we don't need these.

Reply:
I think so...

Reply:
I just checked and we can easily publish any port we want at runtime with `--publish 9981:9981`.
I'm removing all extra ports.